api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -1618,7 +1618,7 @@ public SqlCohortDefinition getPatientsWithSemiAnnualTypeOfDispensation() {
             + "                 INNER JOIN obs o ON e.encounter_id = o.encounter_id"
             + "                 WHERE"
             + "                     e.encounter_type = ${6}"
-            + "                         AND (o.value_coded IN (${1256} , ${1257}) OR o.concept_id = ${23739})"
+            + "                         AND (o.concept_id= 23730 OR o.concept_id = 23739 OR o.concept_id = 23888)"
```

Comment:
@psmatsinhe 
These Concepts should not be indicated directly, please add **${...}**

Reply:
The change was this easy?  I see now the mars journey story from @psmatsinhe